plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/aps/system/services/page/SeoPageMetadata.java

```diff
@@ -156,6 +150,13 @@ public boolean hasEqualConfiguration(Object obj) {
             return check;
         }
         SeoPageMetadata other = (SeoPageMetadata) obj;
+        if (this.getFriendlyCode() == null) {
```

Comment:
Please add a comment on what is happening from 153 to 178. Don't have to change it now but this whole section for this return is really really complex and should be refactored.

Reply:
This method (extension of the core method) allows to understand if the current draft page is different then the related public page. This plugin adds other metadata to the page, so this core method needs to be extended